### ISSUE TITLE
Missing Fonts ( Labels ) on some Modules, some missing graphics

### ISSUE THREAD

**Issue by tamogo** [VCVRack/Rack#399]
OS (if applicable): OSX 10.11.6 El Captain
Version (or "dev" if compiling from source): 0.4.0

Missing Fonts ( Labels ) on some Modules, some missing graphics.
Not on all Modules!
Same issue when rack.app is in the same folder as the jsons 
![screenshot_01](https://user-images.githubusercontent.com/33769389/32975661-dd36a3d4-cc09-11e7-8922-088055ce7dff.png)

**Comment by trummerschlunk** [comment]
same here.
OSX 10.10.5
VCV Rack v0.5.0

**Comment by gideonkiers** [comment]
same here :

NVidia GTX 980Ti
OSX 10.10.5
VCV Rack v0.5.0

**Comment by stannous** [comment]
When using the Nouveau (legacy Nvidia graphics cards (Quadro FX1400, etc.)) I don't get any letters or words when I run Rack on any of my debian/stretch.   But Rack works fine on my other debian servers. and latops.  I thought it might be related to the font i see in the git repo (res/DejaVuSans.ttf) but I think the problem is with the graphics card driver.

**Comment by AndrewBelt** [comment]
This has nothing to do with fonts. Those SVG panels are paths.

**Comment by gideonkiers** [comment]
I installed the same 980ti grphx card in a different machine, with different os (10.12.6) - same rack version (0.5.0) , no problem. I installed a 780ti grphx card in the original machine, same problem, updated to 0.5.1, same problem. So seems to be something with 10.10.x, perhaps in combination with nvidia osx drivers ?

**Comment by stannous** [comment]
What I should have said is this:  I see this same problem when I use nvidia drivers (missing graphics on some modules).  After switching to nouveau drivers, I see an even worse problem (missing graphics AND missing fonts (actually, no fonts at all).  
Using up to date debian stretch with nouveau 1:1.0.13-3 package.

**Comment by tombburnell** [comment]
Any workaround for this?
I'm on a El Capitan Hackintosh X99 with NVIDIA GeForce GTX 980 Ti 6143 MB

**Comment by Leicos** [comment]
Same here on hackintosh  NVIDIA GeForce GTX 970, OS 10.11.6
With Vcv 0.6.0, all modules have this issue

**Comment by Tasm-Devil** [comment]
Same problem here. No text at all.

System: Linux x1 4.17.0-2-MANJARO #1 SMP PREEMPT Fri Jun 8 07:13:17 UTC 2018 x86_64 GNU/Linux


![deepinbildschirmfoto_glfw-application_20180623163600](https://user-images.githubusercontent.com/3172627/41810744-b010c0f6-7703-11e8-9c02-ae16c904f966.png)

**Comment by aghpub** [comment]
@Tasm-Devil Make sure your working directory is the root of the VCV Rack installation. E.g.

`cd <installation directory> && ./Rack`

worked for me.

**Comment by netochka-nezvanova** [comment]
hi 
any solutions for this one?

**Comment by stmllr** [comment]
`cd <installation directory> && ./Rack`

Can you add this into the README, please?

**Comment by AndrewBelt** [comment]
@stmllr  https://vcvrack.com/manual/Installing.html#running-on-linux

**Comment by stmllr** [comment]
@AndrewBelt thanks for the pointer. My idea was a tiny section in the README here at GitHub.

**Comment by AndrewBelt** [comment]
That is beyond the scope of the README for the source code.

**Comment by stmllr** [comment]
I am with you

**Comment by Lacto1** [comment]
Hey, if it can help, i've just solved the non-letters bug on windows 7 with this .dll .. extract the file directly to the rack folder in the C drive.
[OpenGL fix.zip](https://github.com/VCVRack/Rack/files/2886525/OpenGL.fix.zip)


Thx to Nay Seven of the french user group (facebook) .. i was on it since last week hehe
Hope it will help.

**Comment by cello789** [comment]
Just installed vcv rack v1.1.3 and I'm having this same problem on my studio hackintosh running OS X 10.11.6 (el capitan).

Has anyone figured out what causes this? I have an nVidia GTX960sc on the web driver, everything up to date. Just posted to the sub reddit, but then after some more searching found this. I'm new to github, not sure how any of it works...
![Screen Shot 2019-08-11 at 12 45 12 PM](https://user-images.githubusercontent.com/54002690/62837001-f3bd3f80-bc37-11e9-861e-a9f73a84c156.png)

**Comment by lucarautti** [comment]
> Has anyone figured out what causes this? I have an nVidia GTX960sc on the web driver, everything up to date. Just posted to the sub reddit, but then after some more searching found this. I'm new to github, not sure how any of it works...

It's a known issue, OpenGL framebuffer and probably a specific driver incompatibility.
But I don't know if there's a workaround/solution. 
Try to upgrade to macOS 10.12 or 10.13, that is the last OS compatible version with nvidia web drivers.
Have you tried to boot from the integrated Intel GPU?

**Comment by vue-dude** [comment]
Found out the problem is not text but SVG rendering!

**Comment by vue-dude** [comment]
OSX 10.11.6 El Captain
NVIDIA GeForce GTX 960 4095 MB
SVGs rendering fine in preview, affinity designer and all other apps - except for rack!

**Comment by AndrewBelt** [comment]
@vue-dude Yes, the blocky looking labels in this screenshot https://github.com/VCVRack/Rack/issues/399#issue-275042696 are SVG paths, not text elements. Rack's SVG renderer does not support text elements.
It would help to post a screenshot, since your issue is visual.

**Comment by vue-dude** [comment]
Hello Andrew, it looks exactly like the first screenshot in this thread. As SVG rendering and also Open-GL works well on the whole system (tested with cinebench), I guess it could be a minor problem with the NanoVG renderer, maybe just a configuration issue :-)
